src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+}
+
+/* #region stars */
```

Comment:
You can remove regions in your final solution

Reply:
Okay, thank you!